.github/workflows/ci-build-js-ts-test.yml

```diff
@@ -3,8 +3,10 @@ name: CI - JS/TS Tests
 on:
   pull_request:
     paths:
-      - 'browser/javascript/**'
-      - 'browser/typescript/**'
+      - 'browser/javascript/v1/**'
+      - 'browser/typescript/v1/**'
+      - 'browser/javascript/v2/**'
```

Comment:
I think it would be a good idea to have separate workflows for each version so that only the affected examples are retested. We will likely only be updating V2 from now on so it would be nice to not have to wait for v1 tests.

Reply:
It would be a good followup task to separate V1(JS, TS, Node) and V2(JS, TS). We can do this part together with other Ampli SDKs .